frontend/www/js/omegaup/components/notification/Notification.vue

```diff
@@ -55,6 +61,10 @@ export default class Notification extends Vue {
         return ui.formatString(T.notificationNewBadge, {
           badgeName: T[`badge_${this.notification.contents.badge}_name`],
         });
+      case 'demotion':
+        return !!this.notification.contents.message
+          ? this.notification.contents.message
+          : '';
```

Comment:
```suggestion
        return this.notification.contents.message || '',
```

esto es más idiomático.